mle/integration/github.py

```diff
@@ -0,0 +1,222 @@
+import os
+import base64
+import requests
+from datetime import datetime, timezone
+from fnmatch import fnmatch
+
+
+class GithubInte:
```

Comment:
good to merge but I dont like this class name 

GitHubInte is so weird to represent GithubIntegration

But I know naming is so hard~

Reply:
Maybe let's use `GithubIntegration`?

Reply:
I like GitHubIntegration